src/ort_genai.h

```diff
@@ -235,3 +235,8 @@ struct OgaGenerator : OgaAbstract {
 
   static void operator delete(void* p) { OgaDestroyGenerator(reinterpret_cast<OgaGenerator*>(p)); }
 };
+
+struct OgaHandle {
+  OgaHandle() = default;
+  ~OgaHandle() { OgaCheckResult(OgaShutdown()); }
```

Comment:
probably shouldn't throw an exception from the dtor.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c36-a-destructor-must-not-fail

Reply:
Thanks for the reminder. :)